addons/connectToStores.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2015, Yahoo Inc.
+ * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
+ */
+'use strict';
+
+var React = require('react');
+var objectAssign = require('object-assign');
+var contextTypes = require('../lib/contextTypes');
+
+/**
+ * Registers change listeners and retrieves state from stores using `getStateFromStores`
+ * method. Concept provided by Dan Abramov via
+ * https://medium.com/@dan_abramov/mixins-are-dead-long-live-higher-order-components-94a0d2f9e750
+ * @method connectToStores
+ * @param {React.Component} Component component to pass state as props to
+ * @param {array} stores List of stores to listen for changes
+ * @param {object} storeGetters Map of storeName => stateGetterMethod used to retrieve state from the store
+ * @returns {React.Component}
+ */
+module.exports = function connectToStores(Component, stores, storeGetters) {
```

Comment:
React.context does need to be set in some way. `provideContext` at the app level component is probably the best way.